rega-genotype/src/main/java/rega/genotype/ui/viruses/generic/GenericJobOverview.java

```diff
@@ -37,7 +37,9 @@ public GenericJobOverview(GenotypeWindow main, List<ResultColumn> resultColumns)
 			headers.add(new Header(WString.tr("table.header.name")));
 			headers.add(new Header(WString.tr("table.header.length")));
 			headers.add(new Header(WString.tr("table.header.blast")));
-			headers.add(new Header(WString.tr("table.header.phylo"), 2));
+			headers.add(new Header(WString.tr("table.header.phylo"),2));
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file (it doesn't look like the file had any real changes while this push makes it look like it had).